src/Parse/Syntax/FieldParser.php

```diff
@@ -377,17 +377,36 @@ protected function processTagsRegex($string, $tags)
     /**
      * Splits an option string to an array.
      *
-     * one|two           -> [one, two]
-     * one:One|two:Two   -> [one => 'One', two => 'Two']
+     * one|two                -> [one, two]
+     * one:One|two:Two        -> [one => 'One', two => 'Two']
+     * \Path\To\Class::method -> \Path\To\Class::method(): array
      *
      * @param  string $optionsString
+     * @throws Exception
      * @return array
      */
     protected function processOptionsToArray($optionsString)
     {
+        $result = [];
+
+        if (strpos($optionsString, '::') !== false) {
+            $options = explode('::', $optionsString);
+            if (count($options) === 2 && class_exists($options[0]) && method_exists($options[0], $options[1])) {
+                $result = $options[0]::{$options[1]}();
+                if (!is_array($result)) {
+                    throw new Exception(sprintf(
```

Comment:
@JaxWilko 
```suggestion
                    throw new ApplicationException(sprintf(
```

Reply:
Pressed the wrong thing, will make the change in the morning :)